python/lsst/daf/butler/script/queryDatasetTypes.py

```diff
@@ -51,15 +51,14 @@ def queryDatasetTypes(repo, verbose, glob, components):
 
     Returns
     -------
-    collections : `dict` [`str`, [`str`]]
+    collections : `astropy.table.Table`
         A dict whose key is "datasetTypes" and whose value is a list of
         collection names.
     """
     butler = Butler(repo)
-    if not glob:
-        glob = ...
-    datasetTypes = butler.registry.queryDatasetTypes(components=components, expression=glob)
-    info: List[Any]
+    expression = glob if glob else ...
+    datasetTypes = butler.registry.queryDatasetTypes(components=components, expression=expression)
+    info: list[Any]
```

Comment:
`info` does not seem to be used?

Reply:
I didn't write any of this code but I can try to take a quick look to see what's going on.